ui/js/controllers/perf/graphs.js

```diff
@@ -628,11 +628,13 @@ perf.controller('GraphsCtrl', [
         function addSeriesList(partialSeriesList) {
             $scope.loadingGraphs = true;
             return $q.all(partialSeriesList.map(function (partialSeries) {
+                var param = { framework: partialSeries.frameworkId};
+                if (partialSeries.id)
+                    param.id = partialSeries.id;
+                else
+                    param.signature = partialSeriesList.signature;
```

Comment:
I think it is normal these days to use braces for if..else even if there is only one statement. i.e.:

```js
if (foo) {
  ...
} else {
  ...
}
```

Again, there is probably an eslint rule for this somewhere.